stock_scanner/models/scanner_hardware.py

```diff
@@ -469,11 +454,10 @@ def _do_scenario_save(self, message, transition_type, scenario_id=None,
 
                 # No start step found on the scenario, return an error
                 if not step_ids:
-                    return self._send_error(
-                        [_('Please contact'),
-                         _('your'),
-                         _('administrator'),
-                         _('A001')])
+                    return self._send_error([
+                        _('No start step found'),
+                        _('on the scenario'),
```

Comment:
Why are the strings being translated and sent through individually? (genuine question)

Reply:
This is because the very first version of the client didn't wrap and was made for hardware with small screens.
We can concatenate the strings now, thanks :)